ui/src/dashboards/components/GraphOptionsThresholds.js

```diff
@@ -19,8 +19,8 @@ const GraphOptionsThresholds = ({
   onDeleteThreshold,
 }) => {
   return (
-    <div>
-      <label>Thresholds</label>
+    <div className="gauge-controls graph-options-group">
```

Comment:
Is there any way to get references to gauge controls out of graphtable stuff? Or--- could the solution be to name things more generically so it fits both in table graph and gauge controls? We could also take another pass at this when these components are complete. (this work style is an experiment)

Reply:
Yeah I think there's a more generic reusable naming scheme that works for single-stat / gauge / table. I'm most likely going to tackle that closer to the end